malleefowl/processes/wps_workflow.py

```diff
@@ -51,30 +51,58 @@ def __init__(self):
         )
 
     def _handler(self, request, response):
-        def monitor(message, progress):
-            response.update_status(message, progress)
+        # Open the log file from the beginning so that we can write to it any progress
+        with open('logfile.txt', 'w') as fp:
```

Comment:
PyWPS take care of setting the working directory to a temp directory before calling the process handler. See.: https://github.com/geopython/pywps/blob/develop/pywps/app/Service.py (def execute ~line 305)